src/storage.rs

```diff
@@ -1,91 +1,104 @@
-use std::cmp::Ordering;
-use std::collections::BinaryHeap;
-use std::time::{SystemTime, UNIX_EPOCH};
+use std::path::Path;
 
-use telegram_bot::{ChatId, Message, MessageId};
+use rusqlite::{Connection, OpenFlags, Result};
+use telegram_bot::{ChatId, Message as TelegramMessage, MessageId};
+use time::Timespec;
 
-#[derive(Eq)]
-struct Item {
-    chat_id: ChatId,
-    message_id: MessageId,
-    date: u64,
+#[derive(Debug)]
+pub struct Message {
+    id: u32,
+    pub telegram_id: MessageId,
+    pub chat_telegram_id: ChatId,
+    date: Timespec,
```

Comment:
Are you storing dates as strings in sqlite? I think that `INTEGER` is more suitable here. Also, you can use `std::time::Duration` instead of deprecated `time` crate.